app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file is not need it, you are not managing the navigation correctly. The main is the activity which you already had and from there you navigate through the different fragments